Tests/XCUITests/TrackingProtectionTests.swift

```diff
@@ -148,6 +149,6 @@ class TrackingProtectionTests: BaseTestCase {
         // The page is correctly displayed with the lock icon disabled
         XCTAssertTrue(app.staticTexts["This Connection is Untrusted"].exists, "Missing This Connection is Untrusted info")
         XCTAssertTrue(app.staticTexts.elementContainingText("Firefox has not connected to this website.").exists)
-        XCTAssertEqual(app.buttons[AccessibilityIdentifiers.Toolbar.trackingProtection].label, "lockSlashLarge")
+        XCTAssertEqual(app.buttons[AccessibilityIdentifiers.Toolbar.trackingProtection].label, "Connection not secure")
```

Comment:
The test fails because the label is "Secure connection".

Reply:
And in this case it should show "Secure connection" or "Connection not secure"? not sure if there is a bug or we need to update the label...

Reply:
Previously, the label was the name of the image file (`lockSlashLarge`). After #16237, the label should be one from `Strings.swift` instead.

The test fails because the label is currently "Secure connection". 
```
XCTAssertEqual failed: ("Secure connection") is not equal to ("lockSlashLarge")
XCTAssertTrue(app.staticTexts.elementContainingText("Firefox has not connected to this website.").exists) XCTAssertEqual(app.buttons[AccessibilityIdentifiers.Toolbar.trackingProtection].label, "lockSlashLarge") }
/Users/administrator/.jenkins/workspace/firefox-ios-full-functional-ipad/Tests/XCUITests/TrackingProtectionTests.swift:151
```

Related PR diff:
https://github.com/mozilla-mobile/firefox-ios/pull/16237/files#diff-5d47204e2c1e5500f65829619bc1170fe02adf7d8424af700a5c8bb444bc58cbR315

Reply:
My question is, if that `Secure connection` is right, then we will need to update the test to show that instead of `Connection not secure`... if we expect `Connection not secure` and so the test fails because we always see `Secure connection`, I guess there is a bug? otherwise we could update that line too...

It is not clear to me if the PR can be landed as it is or we need some changes, fixes...

Reply:
Yes, I believe that there's a bug on the code, so I opened this PR to demonstrate what the cause may be.

The slash lock icon having the label "Secure connection" seems wrong to me.